src/ctlib/ct.c

```diff
@@ -2142,6 +2142,20 @@ _ct_get_client_type(const TDSCOLUMN *col, bool describe)
 	case SYBUINT8:
 		return CS_UBIGINT_TYPE;
 		break;
+	case SYBUINTN:
```

Comment:
`SYBUINTN` should be filtered out by `tds_get_conversion_type`, for the same reason `SYBINTN` is also not handled in this case.

Reply:
Good point, thanks.  I'd carried this patch forward from an older version that dispatched directly on `tds_type` and handled `SYBINTN` but not `SYBUINTN`.